tests/e2e/kubernetes/kubernetes_test.go

```diff
@@ -433,3 +433,38 @@ func TestKubernetesInstallwithoutRuntimeVersionFlag(t *testing.T) {
 		t.Run(tc.Name, tc.Callable)
 	}
 }
+
+func TestK8sInstallwithoutRuntimeVersionwithMarinerImagesFlag(t *testing.T) {
+	// ensure clean env for test
+	ensureCleanEnv(t, true)
+
+	//	install with mariner images
+	currentVersionDetails.ImageVariant = "mariner"
```

Comment:
reset image variant after this test is over ? Or use a new versionDetails var ...

Reply:
`currentVersionDetails` is reset in `ensureCleanEnv` and it is called before every new sub-test. Should be fine?